Command/ReindexCommand.php

```diff
@@ -219,7 +221,6 @@ private function reindexClass(
                         $this->searchManager->index($object, $locale);
                     }
                     $progress->advance();
-                    $output->write(' Mem: ' . number_format(memory_get_usage()) . 'b');
```

Comment:
i have added this to the progress-bar format which will be printed by symfony 
![image](https://cloud.githubusercontent.com/assets/1464615/15145212/97c99d80-16b4-11e6-8519-0557742e8569.png)